<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="ja-jp">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">9262cd6549f564f0bcc49c7150213f621b9bad3d</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">48cd7791511354f48072998c4b33d3b8cc9251a6</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e5bee60a261b0661b9e5da0d1cf9852a9f6624fa</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown by methods that perform serialization and deserialization.</source>
          <target state="translated">シリアル化および逆シリアル化を実行するメソッドによってスローされる例外。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source><ph id="ph1">&gt; [!NOTE]</ph><ph id="ph2"> &gt;  </ph><ph id="ph3">[!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]</ph>       WorkflowMarkupSerializationException includes information about the line and column locations that caused the exception to be thrown.</source>
          <target state="translated"><ph id="ph1">&gt; [!NOTE]</ph><ph id="ph2"> &gt;  </ph><ph id="ph3">[!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]</ph>WorkflowMarkupSerializationException には例外がスローされる原因となった行と列の位置に関する情報が含まれています。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラスです。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by using the specified error message.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>指定したエラー メッセージを使用してクラスです。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">エラーを説明するメッセージを含む文字列です。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by using serialized data.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラスを使用してデータをシリアル化します。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> that contains the serialized object data about the <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> being thrown.</source>
          <target state="translated">A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>に関する、シリアル化されたオブジェクト データを格納、 <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept>がスローされます。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that contains the context information about the source or destination of the bits the serializer was using.</source>
          <target state="translated">A&lt;xref:System.Runtime.Serialization.StreamingContext&gt;元またはシリアライザーを使用していたビットの転送先に関するコンテキスト情報を格納します&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by using an error message and an exception.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>エラー メッセージと例外を使用してクラスです。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">エラーを説明するメッセージを含む文字列です。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>.</source>
          <target state="translated">例外の原因となった、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>です。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>If <ph id="ph1">`innerException`</ph> is not a null reference (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> in Visual Basic), the current exception is raised in a catch block that handles the inner exception.</source>
          <target state="translated">場合<ph id="ph1">`innerException`</ph>null 参照ではありません (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> Visual Basic で)、内部例外を処理する catch ブロックで現在の例外が発生します。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by using an error message and the line and column numbers that caused the exception to be thrown.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>エラー メッセージと例外がスローされる原因となった行と列の番号を使用してクラスです。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">エラーを説明するメッセージを含む文字列です。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは XOML ファイルをスローする例外の原因となったの行番号。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは例外をスローする原因となった XOML ファイル内の列の位置。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class by using an error message, an exception, and the line and column numbers that caused the exception to be thrown.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>エラー メッセージ、例外、および例外をスローする原因となった行と列の番号を使用してクラスです。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>A string that contains message that describes the error.</source>
          <target state="translated">エラーを説明するメッセージを含む文字列です。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>.</source>
          <target state="translated">例外の原因となった、 <bpt id="p1">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>です。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>If <ph id="ph1">`innerException`</ph> is not a null reference (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> in Visual Basic), the current exception is raised in a catch block that handles the inner exception.</source>
          <target state="translated">場合<ph id="ph1">`innerException`</ph>null 参照ではありません (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> Visual Basic で)、内部例外を処理する catch ブロックで現在の例外が発生します。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは XOML ファイルをスローする例外の原因となったの行番号。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>The column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは例外をスローする原因となった XOML ファイル内の列の位置。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>Sets the specified <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> with information about the <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept>.</source>
          <target state="translated">指定した設定<bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>に関する情報を含む、 <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept>です。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>GetObjectData sets a &lt;xref:System.Runtime.Serialization.SerializationInfo&gt; with all the exception object data targeted for serialization.</source>
          <target state="translated">GetObjectData セット、&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;シリアル化の対象となるすべての例外オブジェクト データ&lt;/xref:System.Runtime.Serialization.SerializationInfo&gt;。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve">
          <source>During de-serialization, the exception is reconstituted from the &lt;xref:System.Runtime.Serialization.SerializationInfo&gt; transmitted over the stream.</source>
          <target state="translated">逆シリアル化中に例外が再構成されて、 &lt;xref:System.Runtime.Serialization.SerializationInfo&gt;、ストリームで転送します&lt;/xref:System.Runtime.Serialization.SerializationInfo&gt;。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> that contains the serialized object data about the <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> being thrown.</source>
          <target state="translated">A <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>に関する、シリアル化されたオブジェクト データを格納、 <bpt id="p2">&lt;xref href="System.Workflow.ComponentModel.Serialization.WorkflowMarkupSerializationException"&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept>がスローされます。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that contains the context information about the source or destination of the bits the serializer was using.</source>
          <target state="translated">A&lt;xref:System.Runtime.Serialization.StreamingContext&gt;元またはシリアライザーを使用していたビットの転送先に関するコンテキスト情報を格納します&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve">
          <source>Gets the line number in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは例外をスローする原因となった XOML ファイル内の行番号を取得します。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve">
          <source>An integer that represents the line number on which the exception was thrown.</source>
          <target state="translated">例外がスローされた行番号を表す整数。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve">
          <source>Gets the column position in the object or XOML file that caused the exception to be thrown.</source>
          <target state="translated">オブジェクトまたは例外をスローする原因となった XOML ファイル内の列の位置を取得します。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve">
          <source>An integer that represents the column position that caused the exception to be thrown.</source>
          <target state="translated">スローする例外の原因となった列の位置を表す整数。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>